tests/template_tests/test_loaders.py

```diff
@@ -10,8 +10,9 @@
 
```

Comment:
For each loader I added tests for ``Loader.get_template`` and ``Loader.load_template_source``. I felt the latter was necessary since otherwise Django would no longer exercise those code paths through the deprecation period.